core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -454,6 +454,11 @@ public void importData() throws IOException, DaoException {
         if( MySQLbulkLoader.isBulkLoad()) {
             MySQLbulkLoader.flushAll();
         }
+        // run sanity check on `mutation_event` to determine whether duplicate
+        // events were introduced during current import
+        if (DaoMutation.hasDuplicateMutationEvents()) {
+            throw new DaoException("Duplicate mutation events were introduced during this import. Aborting...");
```

Comment:
If duplicate mutation events were present before the import and no further duplications were introduced, we would still get this exception. Perhaps it would be more accurate to say:
"After attempting to import mutation data for this study, duplicate mutation event records were detected. Aborting..."